composer.json

```diff
@@ -10,27 +10,35 @@
       "homepage": "https://thephp.foundation"
     }
   ],
+  "repositories": [
+    {
+      "type": "vcs",
+      "url": "https://github.com/pronskiy/dflydev-canal"
+    },
+    {
+      "type": "vcs",
+      "url": "https://github.com/sculpin/sculpin"
+    }
+  ],
   "require": {
-    "php": "^8.0",
-    "sculpin/sculpin": "^3.1",
-    "twig/twig": "2.15.3",
+    "php": "^8.2",
     "ext-gd": "*",
-    "ext-dom": "*"
+    "ext-dom": "*",
+    "twig/twig": "2.15.3",
+    "sculpin/sculpin": "dev-main#467ffd4d7d0e3b28c8b8234d7f29bb3c8caf6c44",
+    "dflydev/canal": "dev-master"
```

Comment:
```suggestion
```

3.0.0-alpha also contain sculpin/sculpin@fc822d9, so I guess we can drop `canal` either